### PR TITLE
Add Philips Hue PAR38 Outdoor Spotlight bulb support

### PR DESCRIPTION
**Pull request by rgregg**
Add support for the new Philips Hue White and Color PAR38 Outdoor Spotlight bulb

### CODE REVIEW

**Comment by Koenkk** [comment]
can you provide a link to a device?

**Comment by rgregg** [comment]
Yep, this is the device: https://www.philips-hue.com/en-us/p/hue-white-and-color-ambiance-par38---e26-smart-bulb/046677577261

File: src/devices/philips.ts [hunk 1]
```diff
@@ -2506,6 +2506,13 @@ const definitions: Definition[] = [
         description: 'Hue Lily outdoor spot light',
         extend: philips.extend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}),
     },
+    {
+        zigbeeModel: ['LCS002'],
+        model: '577262',
```
Comment (Koenkk) [inline src/devices/philips.ts:2511]: When googling on this number, I don't get this bulb, either:

- Use `046677577261` here (EAN)
- Or a model number that is written on the device/box, I expect something in the format of `1742830P7`

Comment (rgregg) [reply src/devices/philips.ts]: Ah thanks, I updated this to match what is on the device itself. In Googling 9290031508, I see a lot of indications that it should have an X at the end (e.g. FCC website), but the device does not have the X on the model number.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**